src/neptune/internal/backends/hosted_artifact_operations.py

```diff
@@ -268,6 +269,34 @@ def get_artifact_attribute(
         raise FetchAttributeNotFoundException(path_to_str(path))
 
 
+@with_api_exceptions_handler
+def get_git_ref_attribute(
```

Comment:
How your horses 🏇, this module is for artifacts only, move it to `hosted_neptune_backend` :wink: